media-video/obs-studio/obs-studio-9999.ebuild

```diff
@@ -44,16 +45,25 @@ COMMON_DEPEND="
 	fdk? ( media-libs/fdk-aac:= )
 	imagemagick? ( media-gfx/imagemagick:= )
 	jack? ( virtual/jack )
+	luajit? ( dev-lang/luajit:2 )
+	nvenc? ( media-video/ffmpeg:=[nvenc] )
 	pulseaudio? ( media-sound/pulseaudio )
+	python? ( ${PYTHON_DEPS} )
 	truetype? (
 		media-libs/fontconfig
 		media-libs/freetype
 	)
 	v4l? ( media-libs/libv4l )
 "
-DEPEND="${COMMON_DEPEND}"
+DEPEND="${COMMON_DEPEND}
+	luajit? ( dev-lang/swig )
+	python? ( dev-lang/swig )
+	speexdsp? ( media-libs/speexdsp )
```

Comment:
I don't see this flag handled by src_configure nor by your patches - how does the flag work?

Reply:
But is it actually build-time automagic? In this case, the existing `pkg_postinst` was wrong anyway. What these messages typically do is notify you about an optional runtime-only dependency.

For build-time automagic, the proper fix would be to unconditionally depend on speexdsp until it is fixed on buildsystem level.

Reply:
Yeah, I only recently noticed that 'speexdsp' needs to be there at build-time for the filter to be available.  Initially I thought it would also be required at run-time, but after testing a bit the filter was still available and working even after uninstalling it, so it would seem like it's build-time only, though optional.

Reply:
in that case I say make it build-time unconditional. dependencies and size are negligible.

Reply:
D'oh.  I'm glad you made me check things again, things I thought I checked properly enough already.

If built with speexdsp support, the 'obs-filters.so' will perhaps unsurprisingly link to 'libspeexdsp.so.1', so it doesn't seem like a build-time only dependency after all.